src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -448,10 +528,16 @@ unique_ptr<CatalogEntry> TableCatalogEntry::ChangeColumnType(ClientContext &cont
 		bound_columns.push_back(COLUMN_IDENTIFIER_ROW_ID);
 	}
 
+	if (columns[change_idx].Generated()) {
+		auto result =
```

Comment:
This line is never run

Reply:
Generated columns cant have their type changed yet, so this never happens, removed it for now